src/main/java/com/google/opensesame/servlets/MentorObject.java

```diff
@@ -6,19 +6,35 @@
 import java.util.List;
 
 public class MentorObject extends PersonObject {
-  private ArrayList<String> projectIDs;
+  private ArrayList<Long> projectIDs;
+  private ArrayList<String> menteeIDs;
 
-  public MentorObject(String gitHubID, ArrayList<String> interestTags, ArrayList<String> projectIDs)
+  public MentorObject(
+      String userID,
+      String gitHubID,
+      ArrayList<String> interestTags,
+      ArrayList<String> menteeIDs,
+      ArrayList<Long> projectIDs,
```

Comment:
No biggie, but one thing to consider here is that in my [ProjectEntity class](https://github.com/googleinterns/step22-2020/blob/master/src/main/java/com/google/opensesame/projects/ProjectEntity.java) I'm storing `projectID`s as strings instead of longs. I'm aware that they come from the GitHub Java API as longs, and I can elaborate on why I chose strings instead of longs. For compatibility's sake I'm wondering if we should discuss, pick one data type, and stick with it.

Reply:
Good to Know, I was under the impression that you were using Longs, I had it set to String earlier. I'll change it back

Reply:
I will take this into account with the UserObject model.

Reply:
Thanks. Sorry this wasn't more clear